pifm.cpp

```diff
@@ -202,7 +203,7 @@ class Outputter : public SampleSink {
     };
     void consume(float* data, int num) {
         for (int i=0; i<num; i++) {
-            float value = data[i]*8;  // modulation index (AKA volume!)
+            float value = data[i]*4*volume;  // modulation index (AKA volume!)
```

Comment:
Well you initially had default volume 8, so technically it should be equivalent to 4*volume and volume defaults to 2.
As that was not that loud on all my systems, I put default to 4 at line 44 and in usage instructions. User can still use 0-8 and 2 will be equivalent, 4 moderate, 0 mute, 8 loud. Above 8 sound starts to break hence range 0-8. 

What is your recommendation?